### DISCUSSION TITLE
Enchant Items Loot Tables

### DISCUSSION THREAD

**Discussion by iwentafk** [OmerBenGera/WildStacker#339] [Denied Feature Requests] [answered]
Hey there,

I'm having issues getting the "enchants" feature to work in the loottable section. I'm trying to get mobs to drop enchanted items, and I'm using the config correctly, but the items being dropped do not contain enchants. I'm not sure what I'm doing wrong.

I'm running **1.15.2**, with WildStacker Version **3.6.0-DEVb217**.

Any help would be great!

Thanks!

**Comment by OmerBenGera** [comment, marked as answer]
You are more than welcome to ask your questions on the discord server so people can help you.
